content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,508 @@
+---
+title: Opening a pull request
+slug: new-content
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 40
+---
+
+{{% capture overview %}}
+
+{{< note >}}
+**Code developers**: If you are documenting a new feature for an
+upcoming Kubernetes release, see
+[Document a new feature](/docs/contribute/new-content/new-features/).
+{{< /note >}}
+
+To contribute new content pages or improve existing content pages, open a pull request (PR). Make sure you follow all the requirements in the [Before you begin](#before-you-begin) section.
+
+If your change is small, or you're unfamiliar with git, read [Changes using GitHub](#changes-using-github) to learn how to edit a page.
+
+If your changes are large, read [Work from a local fork](#fork-the-repo) to learn how to make changes on your locally, on your computer.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Changes using GitHub
+
+If you're less experienced with git workflows, this is an easier method of
+opening a pull request.
+
+1.  On the page where you see the issue, select the pencil icon at the top right.
+    You can also scroll to the bottom of the page and select **Edit this page**.
+
+2.  Make your changes in the GitHub markdown editor.
+
+3.  Below the editor, fill in the **Propose file change**
+    form. In the first field, give your commit message a title. In
+    the second field, provide a description.
+
+    {{< note >}}
+    Do not use any [GitHub Keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in your commit message. You can add those to the pull request
+    description later.
+    {{< /note >}}
```

Comment:
I had a long think on this – the only more visible/central place put this would be in the page's Overview, given the current IA. 

I don't think putting it as a note/warning at the top of the task makes sense either – it's applicable directly to the step about writing commit messages. 

Finally – the "advanced" open a PR topic also has a note along the same lines, but for good measure I changed the wording so it's the same as this one.

It's not ideal, but I don't think there's a better solution.